dhcpv4/option_userclass.go

```diff
@@ -66,6 +66,18 @@ func ParseOptUserClass(data []byte) (*OptUserClass, error) {
 			totalLength, len(data))
 	}
 
+	// Check if option is Microsoft style instead of RFC compliant
+	var counting int
+	for i := 0; i < totalLength; {
+		// UC_Len_i does not include itself so add 1
+		counting += int(data[i]) + 1
+		i += int(data[i]) + 1
```

Comment:
`counting` and `i` look to be the same, can we have only one of them?

Reply:
My bad, fixed